include/algorithms/weak_learner/weak_learner_predict.h

```diff
@@ -69,10 +69,10 @@ namespace interface1
  *      - \ref interface1::Input "Input" class
  *      - \ref interface1::Result "Result" class
  */
-class Batch : public classifier::prediction::Batch
+class Batch : public classifier::prediction::interface1::Batch
```

Comment:
Does it mean that Batch class is deprecated now as there is no interface2::Batch?

Reply:
weak_learner namespace is fully deprecated